en/releases/centreon-core.md

```diff
@@ -19,6 +19,38 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.3
+
+#### Enhancements
+
+- [Core] Implement API log mechanism
+- [Graph] Diverse export size options
+
+#### Bugfixes
+
+- [ACL] Display message if user doesn't have rights to access to a page
```

Comment:
We must remove this ticket, it had a bad fix version and is not fixed in this version